docs/source/metrics.rst

```diff
@@ -33,10 +33,10 @@ The example below shows how to use a metric in your ``LightningModule``:
         self.accuracy = pl.metrics.Accuracy()
 
     def training_step(self, batch, batch_idx):
-        logits = self(x)
+        probs = self(x)
```

Comment:
How did you derive that it has to be probabilities?

Reply:
There is a code like this in Accuracy class
```
    if preds.ndim == target.ndim and preds.is_floating_point():
        # binary or multilabel probablities
        preds = (preds >= threshold).long()
    return preds, target
```

Default threshold == 0.5, so it is probabilities, in case of logits it should be 0.

Reply:
and it also depends on the shape of the tensor, and whether it is binary or multiclass right? these values are unknown in this particular example so it could really be anything :)

Reply:
with multiclass code:
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/metrics/utils.py#L46
```
def _input_format_classification(
    preds: torch.Tensor, target: torch.Tensor, threshold: float = 0.5
) -> Tuple[torch.Tensor, torch.Tensor]:
    """Convert preds and target tensors into label tensors"""

    ...

    if preds.ndim == target.ndim + 1:
        # multi class probabilites
        preds = torch.argmax(preds, dim=1)

    if preds.ndim == target.ndim and preds.is_floating_point():
        # binary or multilabel probablities
        preds = (preds >= threshold).long()
    return preds, target
```

It can be integers (exact predictions), or probabilities [0,1] with default threshold == 0.5. Logits are [-inf, +inf]

Reply:
let's just call it `preds` since it can be anything??

Reply:
Sure, will do, just not logits